.builds/openbsd.yml

```diff
@@ -0,0 +1,34 @@
+image: openbsd/latest
+packages:
+- sqlite3
+- node
+- boehm-gc
+- pcre
+- sfml
+- sdl2
+sources:
+- https://github.com/nim-lang/Nim
+environment:
+  CC: /usr/bin/clang
+tasks:
+- setup: |
+    cd Nim
+    git clone --depth 1 -q https://github.com/nim-lang/csources.git
+    cd csources
+    sh build.sh
+    cd ../
+    bin/nim c --skipUserCfg --skipParentCfg koch
+    sed -i'.original' -e 's/cc = gcc/cc = clang/g' config/nim.cfg
+    echo 'export PATH=$HOME/Nim/bin:$PATH' >> $HOME/.buildenv
+- test: |
+    cd Nim
+    ./koch runCI && exitcode=$? || exitcode=$?
+    if [[ $exitcode -ne 0 ]]; then
+      nim c -r tools/ci_testresults.nim
+    fi
```

Comment:
This should be a cleaner way :P

```sh
    if ! ./koch runCI; then
      nim c -r tools/ci_testresults.nim
      exit 1
    fi
```

Don't worry too much about preserving the exit code, as Nim's tooling doesn't make use of it.

Reply:
Good point, will make that change.